nixos/modules/services/networking/mullvad-vpn.nix

```
@@ -15,6 +15,8 @@ with lib;
   config = mkIf cfg.enable {
     boot.kernelModules = [ "tun" ];
 
+    networking.iproute2.enable = "true";
```

I think the option is a boolean.

```suggestion
    networking.iproute2.enable = true;
```

---

ahh. got k8s manifests on the mind. I'll push the fix now